### PR TITLE
Make pytest-runner an optional dependency

### LINKED ISSUE #54: Installation requires pytest-runner
It would be nice if `setup.py` implemented the conditional requirement suggested by pytest-runner: https://github.com/pytest-dev/pytest-runner#conditional-requirement

### PR DESCRIPTION
**Pull request by jwiggins**
Fixes #54

### CODE REVIEW

**Comment by jwiggins** [comment]
No `set` literals in Python 2.6... TIL

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # -*- coding: utf-8 -*-
 from __future__ import with_statement
+import sys
 
 from setuptools import setup
 
+needs_pytest = set('pytest', 'test', 'ptr').intersection(sys.argv)
```
Comment (sigmavirus24) [inline setup.py]: This needs to be `set(['pytest', 'test', 'ptr'])` :)

Comment (jwiggins) [inline setup.py]: What I get for hurrying... 🙄

**Comment by sigmavirus24** [comment]
🎉

**Pull request merged**